src/module/module.h

```diff
@@ -107,6 +107,13 @@ class Module : public Serialisable<const CoreData &>
     int frequency_;
     // Whether the Module is enabled
     bool enabled_;
+    // Process exit information
+    enum executionResult
+    {
+        failed,
+        success,
+        notExecuted
+    };
```

Comment:
Got my picky hat on today!  Our enum-naming convention is to capitalise the first letter as well. Also we can make this an `enum class` which enforces scope and type better.
```suggestion
    // Module execution result
    enum class ExecutionResult
    {
        Failed,
        Success,
        NotExecuted
    };
```